jvm/src/main/kotlin/app/cash/security_sdk/Certificate.kt

```diff
@@ -0,0 +1,39 @@
+package app.cash.security_sdk
+
+import app.cash.security_sdk.internal.providers.TrifleContentVerifierProvider
+import app.cash.security_sdk.internal.signers.TrifleContentSigner
+import com.squareup.protos.cash.s2dk.api.alpha.Certificate as CertificateProto
+import okio.ByteString
+import okio.ByteString.Companion.toByteString
+import org.bouncycastle.cert.X509CertificateHolder
+import org.bouncycastle.cert.X509v3CertificateBuilder
+import java.math.BigInteger
+import java.time.Instant
+import java.time.Period
+import java.util.Date
+
+/*
+ * Class representing a Trifle certificate.
+ *
+ * @property certificate - bytearray encoding of an x.509 certificate. Subject to change.
+ */
+class Certificate internal constructor(internal val certificate: ByteArray) {
+  /**
+   * Serialize the Trifle Certificate to a ByteArray so clients can store it for inclusion in future
+   * signing operations or provide it as a root certificate to verify signed messages.
+   */
+  fun serialize(): ByteArray {
+    return CertificateProto(certificate = certificate.toByteString(0, certificate.size)).encode()
```

Comment:
nit: `certificate.toByteString()` instead

also since this function is a one liner:

```
fun serialize(): ByteArray = CertificateProto(certificate = certificate.toByteString(0, certificate.size)).encode()
```

Reply:
Done